cmd/virt-handler/virt-handler.go

```diff
@@ -323,11 +347,18 @@ func (app *virtHandlerApp) Run() {
 		}
 	}
 
+	migrationIpAddress := app.PodIpAddress
+	err = findMigrationIP(&migrationIpAddress)
```

Comment:
Maybe let's avoid the pointer magic. How about something like this:


```
findMigrationIP(defaultMigrationIP string) (string, error) {}
```

and then 


```
migrationIPAddress, err := findMigrationIP(app.PodIPAddress)
```